src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,356 @@
+* PayID server: the endpoint that returns payment account(s) information, which conforms to the PayID Protocol.
+* Merchant: Individual or entity receiving a payment (e.g., e-commerce merchant, charity).
+* Payer: Individual or entity originating a payment to a `merchant`.
+* Wallet: A device or application that holds funds (may be a non-custodial wallet).
+* PayID Easy Checkout URL: The URL that is the result of the PayID Easy Checkout protocol; can be used to redirect a client to a wallet corresponding to a particular PayID.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119][] and [RFC9174][].
+
+# Introduction
+
+The PayID Easy Checkout Protocol is a minimal protocol that allows an online merchant to
+request a payment from a payer using only the payer's PayID. Implementations
+of the protocol should require little to no server-side engineering efforts, while creating an improved and uniform
+user experience for payers.
+
+The main focus of the Protocol is on PayID Easy Checkout Discovery, which defines how a PayID client can use a PayID
+to retrieve a PayID Easy Checkout URL which represents a resource that the payer's digital wallet can use to initiate 
+a payment to the merchant. 
+
+Though the [appendix](#appendix) of this specification provides an example usage of a 
+PayID Easy Checkout URL using Web Redirects, supplemental RFCs are needed to define the different ways in which a PayID
+client can utilize a PayID Easy Checkout URL.
+
+# PayID Easy Checkout Protocol
+The PayID Easy Checkout Protocol can be used to facilitate an end-to-end checkout flow between a recipient client, such
+as an online merchant UI, and a sending client, such as a wallet.
+
+The protocol is comprised of two parts:
+
+1. PayID Easy Checkout Discovery
+2. PayID Easy Checkout URL Assembly
+
+## PayID Easy Checkout Discovery
+PayID Easy Checkout extends [PAYID-DISCOVERY][] by defining a new link in the PayID metadata JRD
+returned by a PayID Discovery query. This link, defined in the [JRD section](#payid-easy-checkout-jrds) 
+of this specification, includes the PayID Easy Checkout URL representing a resource on the wallet which can
+be used to complete a payment.
+
+E-commerce merchants who wish to initiate an Easy Checkout flow MUST query the sender's PayID Discovery server to 
+obtain a PayID Easy Checkout URL. Digital wallets and PayID server operators who wish to enable PayID Easy
+Checkout MUST include a JRD Link conforming to the definition in the [JRD section](#payid-easy-checkout-jrds) of this paper 
+in all PayID Easy Checkout Discovery responses.
+
+E-commerce merchants SHOULD implement fallback measures to complete a checkout flow if a user's wallet does not support PayID Easy Checkout.
```

Comment:
```suggestion
Payment receiverrs SHOULD implement fallback measures to complete a checkout flow if a user's wallet does not support PayID Easy Checkout.
```

Reply:
I also wonder if we could just strike this sentence. It feels a bit misplaced, and should be obvious.

Also, line 131 seems to be repeating this instruction.

Reply:
Not sure if it's clear, but I tried to make it clear that EZ Checkout Discovery can fail in two places:
1. If the discovery query returns a non-200 response
2. If the discovery query returns a 200 response, but the JRD doesnt contain an EZ checkout link
I wanted to make sure that the spec suggests fallback measures in either of these cases, but maybe that's unnecessary